### PR TITLE
[Refactor + Fix] : Done some margin,padding fix +refactortion

### PR DESCRIPTION
**Pull request by t007rushi**
minor changes in code base for elegant clean look

### CODE REVIEW

**Review by bossmonkey98 — APPROVED** [review]

File: components/filter.css [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  width: 15rem;
+  width: 13rem;
   display: flex;
   flex-direction: column;
   gap: 1.5rem;
   padding: 0.8rem;
   font-family: sans-serif;
 }
-.filter-wrap {
+.filter-wrapperper {
```
Comment (horsemaker) [inline components/filter.css]: typo?

File: components/filter.css [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
   gap: 1rem;
 }
-.filter-wrap label {
+.filter-wrapperper label {
```
Comment (horsemaker) [inline components/filter.css]: typo?

**Review by horsemaker — APPROVED** [review]

**Review by MarkVed17 — APPROVED** [review]
Just those typos to fix & then you may merge 👍🏼

File: components/filter.css (hunk 1 above)
Comment (t007rushi) [reply components/filter.css]: Thank You So much! That's very much IMP mistake to resolve by me

**Pull request merged**